p2p/protocols/accounting.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The go-ethereum Authors
+// Copyright 2019 The go-ethereum Authors
```

Comment:
If we change the content of the file, should the copyright not be updated? It was actually us who created that file and we are maintaining it

Reply:
We should also change it to `The Swarm Authors`, right?

Reply:
Per @acud's comment I will change the date back to `2018`, but guys: can we come to an agreement on the authorship of this file?

Reply:
We added this file as part of the very first swap commit 8ed4739176f in October 2019. 

I am not familiar at all about these legal things and can't give definitive answer. Because at that point, Swarm was still part of the ethereum code, and if it is wrong to change the copyright date, then probably the whole header should not be changed. I don't know how can answer this in a definitive way. @acud?

Reply:
reverted copyright change.